### PR TITLE
Locations page

### PR DESCRIPTION
**Pull request by franz899**
The current page title is not correct, but is necessary for the page url. I’ll make another PR with more information about it

### CODE REVIEW

File: pages/locations.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+description: Locations
+draft:       false
+layout:      locations.hbs
+title:       locations
```
Comment (james-nash) [inline pages/locations.md]: We should use title case for the page title, so "Locations".

File: pages/locations.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+description: Locations
```
Comment (james-nash) [inline pages/locations.md]: This gets used for the page's meta description, so perhaps put something more descriptive here. Perhaps:

> List of Buildit's global studios with addresses and directions.

File: templates/partials/molecules-location-card.hbs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="grav-c-location-card">
+  <h2>{{name}}</h2>
+  <p>
+    {{#if addressline1}}{{addressline1}}<br>{{/if}}
+    {{#if addressline2}}{{addressline2}}<br>{{/if}}
+    {{#if city}}{{city}}<br>{{/if}}
+    {{#if postcode}}{{postcode}}<br>{{/if}}
+    {{#if country}}{{country}}{{/if}}
+  </p>
+  <a href="{{map-link}}" title="See {{name}} office on Google Maps - Opens in a new tab" target="_blank">
```
Comment (james-nash) [inline templates/partials/molecules-location-card.hbs]: 😭

File: pages/locations.md (hunk 1 above)
Comment (franz899) [reply pages/locations.md]: Not that quickly, the title is going to change, like I said in the description. The page title is not suited to be used at permalink.

File: pages/locations.md (hunk 2 above)
Comment (franz899) [reply pages/locations.md]: Yea, that one is better.

**Pull request merged**